uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -26,6 +26,7 @@ define([
       all: function (filters, pagination, sort, params, cache) {
         $log.debug('LeaveRequestAPI.all');
 
+        sort = sort || 'from_date DESC';
```

Comment:
i would keep the api service free of "business logic". I don't see any benefit of enforcing the sorting by from_date by default, let the controller ask for that